std/ndn/spec.go

```diff
@@ -42,12 +42,21 @@ type Interest interface {
 
 // InterestConfig is used to create a Interest.
 type InterestConfig struct {
+	// Standard Interest parameters
 	CanBePrefix    bool
 	MustBeFresh    bool
 	ForwardingHint []enc.Name
 	Nonce          *uint64
 	Lifetime       *time.Duration
 	HopLimit       *uint
+
+	// Signed Interest parameters
```

Comment:
If you have time, please add some comments explaining that we are reducing the usage of signed Interests, and that's why we want to decouple these parameters from `Signer`s.

Reply:
Done.